opal/corelib/kernel.rb

```diff
@@ -35,11 +35,9 @@ def <=>(other)
   def method(name)
     %x{
       var meth = self['$' + name];
-
       if (!meth || meth.$$stub) {
-        #{raise NameError.new("undefined method `#{name}' for class `#{self.class}'", name)};
+        #{::Kernel.raise NameError.new("undefined method `#{name}' for class `#{self.class}'", name)};
```

Comment:
This shouldn't be necessary as `#raise` is also implemented by `Kernel` so it should be already available in this context.